spec/mobility/plugins/active_model/dirty_spec.rb

```diff
@@ -284,7 +284,7 @@ def save
         expect(article.title).to eq("Title")
         expect(article.changed?).to eq(true)
         expect(article.changed).to match_array(["title_ja", "title_en"])
-        expect(article.changes).to eq({ "title_ja" => [nil, "ばばば"], "title_en" => ["ばばば", "Title"]})
+        expect(article.changes).to eq({ "title_ja" => [nil, "ばばば"], "title_en" => [nil, "Title"]})
```

Comment:
As a byproduct, this change also makes dirty/fallbacks compatibility a bit less weird.